sbt-dotty/sbt-test/sbt-dotty/compiler-plugin/build.sbt

```diff
@@ -7,13 +7,14 @@ lazy val pluginSetting = Seq(
   scalaVersion := dottyVersion,
 
   libraryDependencies ++= Seq(
-    "ch.epfl.lamp" %% "dotty" % scalaVersion.value % "provided"
+    "ch.epfl.lamp" %% "dotty-compiler" % scalaVersion.value % "provided"
   )
 )
 
 lazy val plugin = (project in file("plugin")).settings(pluginSetting: _*)
 
-lazy val app = (project in file(".")).settings(
-  scalaVersion := dottyVersion,
-  libraryDependencies += compilerPlugin("ch.epfl.lamp" %% "dividezero" % "0.0.1")
-)
+scalaVersion := dottyVersion
```

Comment:
There's already a scalaVersion set above.

Reply:
The `scalaVersion` above is for a different sub-project.

Reply:
oh I see yes, I suggest explicitly using `lazy val root = (project in file("."))` when there's more than one project, I find it less confusing.

Reply:
Yes, good advice, that's less confusing.